docs/sql-ref-syntax-aux-describe-table.md

```diff
@@ -62,6 +63,58 @@ to return the metadata pertaining to a partition or column respectively.
     
     **Syntax:** `[ database_name. ] [ table_name. ] column_name`
 
+* **AS JSON**
+
+  An optional parameter to return the table metadata in JSON format. Only supported when EXTENDED
+  format is specified.
+
+  **Syntax:** `[ AS JSON ]`
+
+  **Schema:**
```

Comment:
Do we have a different JSON schema for describing a single column?

Reply:
yes, but for this PR we won't support JSON format for single column and for now will throw `QueryCompilationErrors.describeColJsonUnsupportedError()`. Also added this to the docs page